osu.Game/Screens/Select/Leaderboards/LeaderboardScore.cs

```diff
@@ -0,0 +1,378 @@
+                                    Children = new Drawable[]
+                                    {
+                                        nameLabel = new OsuSpriteText
+                                        {
+                                            Text = Score.User?.Username ?? Score.Username,
+                                            Font = @"Exo2.0-BoldItalic",
+                                            TextSize = 23,
+                                        },
+                                        new FillFlowContainer
+                                        {
+                                            Origin = Anchor.BottomLeft,
+                                            Anchor = Anchor.BottomLeft,
+                                            AutoSizeAxes = Axes.Both,
+                                            Direction = FillDirection.Horizontal,
+                                            Spacing = new Vector2(10f, 0f),
+                                            Children = new Drawable[]
+                                            {
+                                                flagBadgeContainer = new Container
+                                                {
+                                                    Size = new Vector2(87f, 20f),
+                                                    Masking = true,
+                                                    Children = new Drawable[]
+                                                    {
+                                                        flag,
+                                                    },
+                                                },
+                                                new FillFlowContainer
+                                                {
+                                                    AutoSizeAxes = Axes.Both,
+                                                    Direction = FillDirection.Horizontal,
+                                                    Spacing = new Vector2(10f, 0f),
+                                                    Margin = new MarginPadding { Left = 10, },
+                                                    Children = new Drawable[]
+                                                    {
+                                                        maxCombo = new ScoreComponentLabel(FontAwesome.fa_link, Score.MaxCombo.ToString()),
+                                                        accuracy = new ScoreComponentLabel(FontAwesome.fa_crosshairs, string.Format(Score.Accuracy % 1 == 0 ? @"{0:0}" : @"{0:0.00}", Score.Accuracy)),
+                                                    },
+                                                },
+                                            },
+                                        },
+                                    },
+                                },
+                                scoreRank = new DrawableRank(Score.Rank)

```

Comment:
`scoreRank` and `scoreLabel` should probably be in a flow? will avoid the `Position = new Vector2(-score_rank_size - 5f, 0f),` nonsense below.